R/pkg/NAMESPACE

```diff
@@ -10,6 +10,7 @@ export("sparkR.session")
 export("sparkR.init")
 export("sparkR.stop")
 export("sparkR.session.stop")
+export("conf")
```

Comment:
sure - its dependency on sparkSession - i think it belongs to SQLContext.R, should it be `sparkR.session.conf` (because it is the session's RuntimeConfig) but it feels too long.